recipes/django-mfa2/LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Please merge/rebase from master

Reply:
merged from master and fixed license file

Reply:
The license file still seems empty...

Reply:
So it would seem. I apologize. It didn't commit for some reason.